Modules/Devices/src/Devices.Infrastructure/Persistence/Repository/OAuthClientsRepository.cs

```diff
@@ -17,7 +20,30 @@ public OAuthClientsRepository(OpenIddictApplicationManager<OpenIddictEntityFrame
     public async Task<IEnumerable<OAuthClient>> FindAll(CancellationToken cancellationToken)
     {
         var clients = await _applicationManager.ListAsync(applications => applications.Select(c => new OAuthClient(c.ClientId, c.DisplayName)), cancellationToken).ToListAsync(cancellationToken);
-
         return clients;
     }
+
+    public async Task<OAuthClient> Find(string clientId, CancellationToken cancellationToken)
+    {
+        var client = await _applicationManager.FindByClientIdAsync(clientId, cancellationToken);
+        return client != null ? new OAuthClient(client.ClientId, client.DisplayName) : null;
+    }
+
+    public async Task Add(string clientId, string displayName, string clientSecret, CancellationToken cancellationToken)
+    {
+        var managerResult = await _applicationManager.CreateAsync(new OpenIddictApplicationDescriptor
```

Comment:
The result is actually a client, so you should call it like that (=> `client`). 

I have no idea why it was called managerResult. Doesn't sound like a name I would come up with. So it probably was the person who made the OpenIddict change.